mousestyles/dynamics/__init__.py

```diff
@@ -337,3 +337,92 @@ def mcmc_simulation(mat_list, n_per_int):
             simu_result[index] = int(state_j)
     simu_result = np.array(simu_result, dtype=int)
     return simu_result
+
+
+def get_score(true_day, simulated_day, weight=[1, 10, 50, 1]):
```

Comment:
Could it be better to use weights that are regularized, so that each weight is between 0 and 1 and the sum of all weights is 1?